test/test_common/network_utility.cc

```diff
@@ -105,9 +105,7 @@ const std::string addressVersionAsString(const Address::IpVersion version) {
 Address::InstanceConstSharedPtr getSomeLoopbackAddress(Address::IpVersion version) {
   if (version == Address::IpVersion::v4) {
     // Pick a random address in 127.0.0.0/8.
-    // TODO(jamessynge): Consider how to use $GTEST_RANDOM_SEED for seeding the rng.
-    // Perhaps we need a TestRuntime::getRandomGenerator() or similar.
-    Runtime::RandomGeneratorImpl rng;
+    TestRandomGenerator rng;
```

Comment:
Something like the static const int32_t SEED in utility.cc?  :-)

I've verified that test/common/network/listener_impl_test uses a common seed for all 12 RNGs, even with a bonus sleep(1) thrown in to ensure time_since_epoch() varies.